extensions/debuggee/src/org/exist/debuggee/ScriptRunner.java

```diff
@@ -69,7 +69,7 @@ public void run() {
 		try {
 			final Database db = BrokerPool.getInstance();
 			
-            BrokerPool.registerStatusObserver(this);
+//            BrokerPool.registerStatusObserver(this);
```

Comment:
AFAIK the Debuggee code does not work and is unsupported, which is why I did not spend too much time on that, am I mistaken?

If that is not the case, then each instance of BrokerPool now has a `status observer`, we could use that here if you like?